german.feskov/task-2-2/internal/heapmax/heapmax.go

```diff
@@ -0,0 +1,41 @@
+package heapmax
+
+import "errors"
+
+var errPushCast = errors.New("bad cast to int")
+
+type IntHeap []int
+
+func (h *IntHeap) Len() int {
+	return len((*h))
+}
+
+func (h *IntHeap) Less(i, j int) bool {
+	return (*h)[i] > (*h)[j]
+}
+
+func (h *IntHeap) Swap(i, j int) {
+	(*h)[i], (*h)[j] = (*h)[j], (*h)[i]
+}
+
+func (h *IntHeap) Push(x any) {
+	v, ok := x.(int)
+	if !ok {
+		panic(errPushCast)
```

Comment:
вот тут статическая ошибка не нужна...


```suggestion
		panic("bad cast to int")
```